query/workmanager_test.go

```diff
@@ -384,7 +386,10 @@ func TestWorkManagerWorkRankingScheduling(t *testing.T) {
 	const numQueries = 4
 	const numWorkers = 8
 
-	wm, workers := startWorkManager(t, numWorkers)
+	workMgr, workers := startWorkManager(t, numWorkers)
+
+	wm, ok := workMgr.(*workManager)
```

Comment:
Alternatively looks like we can just modify `startWorkManager` to return a pointer to the struct itself?